### PR TITLE
Added example for barcode plugin

### PR DESCRIPTION
**Pull request by SergeoLacruz**

### CODE REVIEW

File: docs/docs/report/report.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ InvenTree supports the following reporting functionality:
 | [Purchase Order Report](./purchase_order.md) | Format a purchase order report |
 | [Sales Order Report](./sales_order.md) | Format a sales order report |
 | [Return Order Report](./return_order.md) | Format a return order report |
+| [Stock Location Report](./stocl_location.md) | Format a stock location report |
```
Comment (SchrodingersGat) [inline docs/docs/report/report.md:151]: Looks like a typo here?

Comment (SergeoLacruz) [reply docs/docs/report/report.md]: Thanks for the hint. Fixed that.

File: docs/docs/extend/plugins/barcode.md [hunk 2]
```diff
@@ -23,3 +23,35 @@ POST {
...
+
+class InvenTreeBarcodePlugin(BarcodeMixin, InvenTreePlugin):
+
+    NAME = "MyBarcode"
+    TITLE = "My Barcodes"
+    DESCRIPTION = "support for barcodes"
+    VERSION = "0.0.1"
+    AUTHOR = "Michael"
+
+    Status=0
```
Comment (wolflu05) [inline docs/docs/extend/plugins/barcode.md:45]: I would stick to conventions here and use lowercase snake case for variables. And secondly I would add some spacing around the =. (Also at other lines)
```suggestion
    status = 0
```

Comment (SchrodingersGat) [reply docs/docs/extend/plugins/barcode.md:45]: @SergeoLacruz please address this as suggested by @wolflu05 then happy to merge

**Comment by SchrodingersGat** [comment]
@SergeoLacruz thanks for the contributino

**Pull request merged**